maven-plugins/helidon-archetype-maven-plugin/src/it/projects/test7/postbuild.groovy

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+def actualIndex = 0
+def actualLines = new File(basedir, "build.log").readLines()
+
+
+def findLines(actualIndex, actualLines, fname) {
```

Comment:
Can you use `JUnitLauncher` in the groovy file and implement the test in Java. See the the groovy files from `1..4`. It seems that current test 5 and 6 did not do this they need to be updated as well (see https://github.com/helidon-io/helidon-build-tools/issues/931).